src/pages/Login/Login.scss

```
@@ -0,0 +1,88 @@
+.login {
+  display: flex;
+  align-items: center;
+  height: 100vh;
+
+  main {
+    width: 280px;
+    margin: 0 auto;
+
+    h2 {
+      margin-bottom: 30px;
+      font-size: 20px;
+      font-weight: 600;
+      text-align: center;
+    }
+
+    form {
+      display: flex;
+      flex-direction: column;
+
+      input {
+        padding: 10px;
+        border: 1px solid #cccccc;
+        border-radius: 3px;
+
+        &::placeholder {
+          color: #cccccc;
+          font-size: 10px;
+        }
+
+        &:first-child {
+          margin-bottom: 10px;
+        }
+      }
+    }
+
+    .loginSub {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 15px 0 30px 0;
```

다음과 같이 축약해서 작성하실 수 있습니다.
(https://developer.mozilla.org/ko/docs/Web/CSS/padding)

```suggestion
      padding: 15px 0 30px;
```

---

반영완료했습니다:)